ParkBlazerApp/src/pages/UserReports.tsx

```diff
@@ -0,0 +1,66 @@
+import { IonPage, IonGrid, IonRow, IonCol, IonCard, IonCardHeader, IonCardTitle, IonCardContent, IonList, IonItem, IonThumbnail, IonLabel, IonText, IonContent } from "@ionic/react";
+import Navbar from "../components/navbar";
+import { useState } from "react";
+import { useEffect } from "react";
+import axios from "axios";
+import AuthService from "../utils/AuthService";
+import "./UserReports.css";
+import { formatDistanceToNow } from "date-fns";
+import { de } from "date-fns/locale";
+
+const UserReports: React.FC = () => {
+    const [reports, setReports] = useState<any>();
+
+    useEffect(() => {
+        fetchAvailabilityReports();
+    }, []);
+
+    const fetchAvailabilityReports = async () => {
+        const token = AuthService.getToken();
+        try {
+            const response = await axios.get('https://server-y2mz.onrender.com/api/user_reports', {
+                headers: {
+                    Authorization: `Bearer ${token}`
+                }
+            });
+            const reportsData = response.data;
+            setReports(reportsData.reports);
+            console.log(reports)
+        } catch (error) {
+            console.error("Fehler beim Abrufen der Parkplatzverfügbarkeitsberichte:", error);
+        }
+    };
```

Comment:
Diese Methode sollte auch in ``./data/reports.ts``

Reply:
Hast recht, aber schaffe ich leider zeitlich nicht mehr...